src/containers/Tenant/utils/schemaActions.ts

```diff
@@ -37,11 +38,11 @@ const bindActions = (
     dispatch: React.Dispatch<any>,
     additionalEffects: ActionsAdditionalEffects,
 ) => {
-    const {setActivePath, setQueryMode, showCreateDirectoryDialog} = additionalEffects;
+    const {setActivePath, setQueryExecutionSettings, showCreateDirectoryDialog} = additionalEffects;
 
     const inputQuery = (tmpl: (path: string) => string, mode?: QueryMode) => () => {
         if (mode) {
-            setQueryMode(mode);
+            setQueryExecutionSettings({...DEFAULT_QUERY_SETTINGS, queryMode: mode});
```

Comment:
It should be current settings, not default, only mode is changed.

Also there is a bug with settings that changed this way - although they are changed, old settings displayed in the dialog

How to reproduce:
1) Set not default settings, save them, gear will show 5 and will be blue
2) Select 'Create table' action of DB or Dir in schema tree
3) There will be no blue gear or changes sign, settings will be default, but if you open dialog, old settings will be displayed there 

<img width="623" alt="Screenshot 2024-07-30 at 13 03 34" src="https://github.com/user-attachments/assets/ba809a84-8dbb-4f78-bbb0-a7b0ecb011db">

Reply:
need to explicitly reset dialog form values on settings change
added useEffect